control/autoware_simple_pure_pursuit/src/simple_pure_pursuit.cpp

```diff
@@ -113,10 +113,12 @@ autoware_control_msgs::msg::Lateral SimplePurePursuitNode::calc_lateral_control(
     lookahead_gain_ * target_longitudinal_vel + lookahead_min_distance_;
 
   // calculate center coordinate of rear wheel
-  const double rear_x = odom.pose.pose.position.x -
-                        vehicle_info_.wheel_base_m / 2.0 * std::cos(odom.pose.pose.orientation.z);
-  const double rear_y = odom.pose.pose.position.y -
-                        vehicle_info_.wheel_base_m / 2.0 * std::sin(odom.pose.pose.orientation.z);
+  const double rear_x =
+    odom.pose.pose.position.x -
+    vehicle_info_.wheel_base_m / 2.0 * std::cos(tf2::getYaw(odom.pose.pose.orientation));
+  const double rear_y =
+    odom.pose.pose.position.y -
+    vehicle_info_.wheel_base_m / 2.0 * std::sin(tf2::getYaw(odom.pose.pose.orientation));
```

Comment:
LGTM, but just as a suggestion, consider saving the yaw value as a variable so that we don't have to recompute it from quaternion.

```suggestion
  const double vehicle_yaw = tf2::getYaw(odom.pose.pose.orientation);
  const double rear_x =
    odom.pose.pose.position.x -
    vehicle_info_.wheel_base_m / 2.0 * std::cos(vehicle_yaw));
  const double rear_y =
    odom.pose.pose.position.y -
    vehicle_info_.wheel_base_m / 2.0 * std::sin(vehicle_yaw));
```

Reply:
Good suggestion Mits san， @mitsudome-r 

I have  include your commit in latest rebase

Thanks

心刚